Mathlib/Data/Nat/GCD/BigOperators.lean

```diff
@@ -30,4 +30,38 @@ theorem coprime_prod_right {ι : Type*} {x : ℕ} {s : ι → ℕ} {t : Finset 
   Finset.prod_induction s (fun y ↦ x.Coprime y) (fun a b ↦ Coprime.mul_right) (by simp)
```

Comment:
Could you move these two lemmas below and shorten their proofs?

Reply:
I think using `alias` would be appropriate here, and perhaps renaming to `Coprime.prod_left` etc.

Reply:
I redefined using alias and renamed to `Coprime.prod_left` and `Coprime.prod_right`.